### ISSUE TITLE
1.17.1: Can teleport along same elevation, and above but not down

### ISSUE THREAD

**Issue by bentwookie** [jrbudda/minecrift#344]
I'm running 1.17.1 with a Valve Index in standing mode. I can teleport along the same level of blocks and above, but never down. Not even one block down where there wouldn't be fall damage. 

I don't think I'm using Forge (forgive me for not knowing exactly. I'm middle aged ;) )

**Comment by bentwookie** [comment]
I got up out of my chair and see I'm able to walk forward with the sticks, so maybe this is as designed? 

BTW, this is really cool. Super fun to see my kid come across a golem and shout, THESE ARE MUCH BIGGER IN HERE!

**Comment by GreatSalamander** [comment]
I get the same issue. I notice it will sometimes let me teleport down if I'm jumping but not while standing still.

Edit: I think it's because the game thought I was crouched? I was playing in standing mode sitting down, so that was probably why.

**Comment by fayer3** [comment]
are you maybe crouching? the game prevents teleporting down when crouching.

**Comment by GreatSalamander** [comment]
> are you maybe crouching? the game prevents teleporting down when crouching.

Yes I was playing while sitting down in standing mode so the game assumed I was crouching. I'll mess with the options to fix the issue, sorry for the trouble.

**Comment by Techjar** [comment]
http://www.vivecraft.org/faq/#lazybutt
